modin/core/io/io.py

```diff
@@ -652,7 +652,7 @@ def to_csv(cls, obj, **kwargs):  # noqa: PR01
     @_inherit_docstrings(
         pandas.DataFrame.to_parquet, apilink="pandas.DataFrame.to_parquet"
     )
-    def to_parquet(cls, obj, **kwargs):  # noqa: PR01
+    def to_parquet(cls, obj, path, **kwargs):  # noqa: PR01
```

Comment:
How did this work before?

Reply:
This parameter was passed by keyword (the code worked before), now as a positional one, so that there are no conflicts between several names `filename_or_buffer` and `path`.